ffmpeg

```diff
@@ -1 +1 @@
-Subproject commit 8dd84dc1ea8bc70da1a345109383f62974a9c2fc
+Subproject commit dd6d6a216d5411dbd5bbe902c2d529904184fc9d
```

Comment:
Yeah it was difficult to work out the issue and then the git moved forward and a lot of the code became outdated.